server/game/conflict.js

```diff
@@ -20,11 +20,15 @@ class Conflict {
         this.elements = [];
         this.attackers = [];
         this.attackerSkill = 0;
-        this.attackerSkillModifier = 0;
+        this.attackerMilitarySkillModifier = 0;
+        this.attackerPoliticalSkillModifier = 0;
```

Comment:
I 100% agree, but I didn't want to rip them out since that wasn't specifically my task here and I didn't know if there was going to be a use case one day